### PR TITLE
Fix rubocop rules

### PR DESCRIPTION
**Pull request by archanaserver**

### CODE REVIEW

File: foreman_puppet.gemspec [hunk 1]
```diff
@@ -13,4 +13,6 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,db,lib,locale,webpack}/**/*'] + ['LICENSE', 'Rakefile', 'README.md', 'package.json']
   s.test_files = Dir['test/**/*']
+
+  s.required_ruby_version = '>= 2.7', '< 4'
```
Comment (stejskalleos) [inline foreman_puppet.gemspec:17]: Why `< 4`? It should be just `~> 2.7.0`

File: .rubocop.yml [hunk 2]
```diff
@@ -8,6 +8,12 @@ inherit_mode:
   merge:
     - Exclude
 
+AllCops:
+  TargetRubyVersion: '2.7'
+  Exclude:
+    - 'test/**/*'
```
Comment (ekohl) [inline .rubocop.yml:14]: I don't think it's good to exclude RuboCop on all tests.

File: foreman_puppet.gemspec (hunk 1 above)
Comment (ekohl) [reply foreman_puppet.gemspec:17]: `~> 2.7.0` would prevent it from running on Ruby 3, which is something we'd like to do.

**Review by ekohl — APPROVED** [review]
Lgtm. You could argue we want to first test Ruby 3, but without the line it'd also install on Ruby 3. I'd go ahead with this but leaving to @stejskalleos for a final approval

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

**Pull request merged**